llvm/tools/llvm-objcopy/llvm-objcopy.cpp

```diff
@@ -67,12 +67,18 @@ using namespace llvm::object;
 // The name this program was invoked as.
 static StringRef ToolName;
 
-static ErrorSuccess reportWarning(Error E) {
+namespace llvm {
+namespace objcopy {
+
+ErrorSuccess reportWarning(Error E) {
```

Comment:
Please follow the guideline described here: https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions

Reply:
Done.